beater/handlers.go

```diff
@@ -263,7 +263,7 @@ func ipRateLimitHandler(rateLimit int, h http.Handler) http.Handler {
 	}
 
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if deny(utility.ExtractIP(r)) {
+		if deny(utility.RemoteAddr(r)) {
```

Comment:
Could you also change https://github.com/elastic/apm-server/pull/927/files#diff-0ee6b4dd463eb95280cc3653bd4701b6L227 to `utility.RemoteAddr` ?

Reply:
Oops, I thought I did - must have reverted it by accident. Pushed a new commit. If you're merging, can you please squash? I haven't done it, as I don't want to mess up the review history.